### DISCUSSION TITLE
Adding an overlay to the skybox, or have scalable image hotspots

### DISCUSSION THREAD

**Discussion by damienshiells** [google/model-viewer#4660] [Q&A]
I have a skybox of a room with a desk on it. I'd like to add desk items to it (e.g. a lamp, laptop, etc). I can't see a way to overlay images on the skybox, and I tried an image as a hotspot but it doesn't scale the same as the skybox when I change the field of view.

Is there a way to achieve what I am trying to do?

Thanks in advance.

**Comment by danilokrulj** [comment]
I think you will have to edit the skybox in Photoshop to achieve something like that.

**Reply by damienshiells** [reply]
The items on the desk will be dynamic, so having a different skybox for each combination isn't feasible.

Are you aware of a way to "scale" the hotspots on zoom/field of view change? Not sure if it's possible.

**Reply by danilokrulj** [reply]
I don't think there's a ready to go solution for that but I'm guessing you could theoretically catch the FOV change and change the scale of the hotspots accordingly? I'm not sure how that would work, it's just a theory.

**Reply by damienshiells** [reply]
Yeah I did think of that, hoping to avoid it, it feels hacky.

I'll try and give it a go. I appreciate you getting back to me!

**Reply by elalish** [reply]
Perhaps an easier solution would be to disable the FoV change entirely? Just set `max-field-of-view` to the same value as `min-field-of-view`.

**Reply by damienshiells** [reply]
Thanks for the suggestion @elalish but I do need that functionality.
